lib/cext/ruby.h

```diff
@@ -101,11 +101,11 @@ VALUE rb_tr_hidden_variable_get(VALUE object, const char *name);
 
 // Handles
 
-void *rb_tr_handle_for_managed(void *managed);
-void *rb_tr_handle_for_managed_leaking(void *managed);
-VALUE rb_tr_managed_from_handle_or_null(void *handle);
-void *rb_tr_managed_from_handle(void *handle);
-void rb_tr_release_handle(void *handle);
+void* rb_tr_handle_for_managed(VALUE managed);
+void* rb_tr_handle_for_managed_leaking(VALUE managed);
```

Comment:
Why did you make this change? This is the opposite of what we do everywhere else - the `*` is part of the name, not the type, in C.